unit/solvers/smt2_incremental/object_tracking.cpp

```diff
@@ -12,41 +12,12 @@
 #include <string>
 #include <utility>
 
-TEST_CASE("find_object_base_expression", "[core][smt2_incremental]")
```

Comment:
@thomasspriggs is going to re-add a fixed version of this test after this PR has been merged.